src/main/java/com/hubspot/maven/plugins/dependency/management/DependencyManagementAnalyzer.java

```diff
@@ -187,4 +188,13 @@ private RequireManagmentConfig getEffectiveRequireManagementConfig(String key) {
 
     return dependencyMap;
   }
+
+  private void logMessage(String message) {
```

Comment:
Maybe call this something like `logViolation` (`message` implies it might be at info).